docs/source/workflow_inputs.rst

```diff
@@ -4063,6 +4065,7 @@ Each heat pump water heater is entered as a ``/HPXML/Building/BuildingDetails/Sy
   .. [#] The sum of all ``FractionDHWLoadServed`` (across all WaterHeatingSystems) must equal to 1.
   .. [#] FractionDHWLoadServed represents only the fraction of the hot water load associated with the hot water **fixtures**.
          Additional hot water load from clothes washers/dishwashers will be automatically assigned to the appropriate water heater(s).
+  .. [#] If HeatingCapacity not provided, defaults to 1706 Btu/hr (0.5 kW) multiplied by the heat pump COP.
```

Comment:
@jmaguire1 At some point, let's review if this is still a good assumption. I'm adding it to https://github.com/NREL/OpenStudio-HPXML/issues/1170.